vips/vip-408/bsctestnet.ts

```diff
@@ -0,0 +1,197 @@
+        params: [sepolia.POOL_REGISTRY, parseUnits("10000", 6)],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: VUSDC_Ethena,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["7200"],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            VUSDC_Ethena,
+            parseUnits("0", 18),
+            parseUnits("0", 18),
+            parseUnits("10000", 6),
+            sepolia.VTREASURY,
+            parseUnits("50000000", 6),
+            parseUnits("46000000", 6),
+          ],
+        ],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: VsUSDe_Ethena,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [parseUnits("0.010", 18)],
+        dstChainId: LzChainId.sepolia,
+      },
+
+      {
+        target: VUSDC_Ethena,
+        signature: "setProtocolSeizeShare(uint256)",
+        params: [parseUnits("0.020", 18)],
+        dstChainId: LzChainId.sepolia,
+      },
+
+      {
+        target: COMPTROLLER_ETHENA,
+        signature: "setActionsPaused(address[],uint8[],bool)",
+        params: [[VsUSDe_Ethena, VUSDC_Ethena], [2, 7], true],
+        dstChainId: LzChainId.sepolia,
+      },
+
+      // Configure converters
+
+      ...Object.entries(converterBaseAssets).map(([converter, baseAsset]: [string, string]) => ({
+        target: converter,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [baseAsset, [MockPT_USDe_27MAR2025], [[CONVERSION_INCENTIVE, ConversionAccessibility.ALL]]],
+        dstChainId: LzChainId.sepolia,
+      })),
+
+      ...Object.entries(converterBaseAssets).map(([converter, baseAsset]: [string, string]) => ({
+        target: converter,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [baseAsset, [MockPT_sUSDE_27MAR2025], [[CONVERSION_INCENTIVE, ConversionAccessibility.ALL]]],
+        dstChainId: LzChainId.sepolia,
+      })),
+
+      ...Object.entries(converterBaseAssets).map(([converter, baseAsset]: [string, string]) => ({
+        target: converter,
+        signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+        params: [baseAsset, [MocksUSDe], [[CONVERSION_INCENTIVE, ConversionAccessibility.ALL]]],
+        dstChainId: LzChainId.sepolia,
+      })),
+
+      ...Object.entries(converterBaseAssets)
+        .filter(([, baseAsset]) => baseAsset !== MockUSDC) // Skip if base asset is USDC
+        .map(([converter, baseAsset]: [string, string]) => ({
+          target: converter,
+          signature: "setConversionConfigs(address,address[],(uint256,uint8)[])",
+          params: [baseAsset, [MockUSDC], [[CONVERSION_INCENTIVE, ConversionAccessibility.ALL]]],
+          dstChainId: LzChainId.sepolia,
+        })),
```

Comment:
AFAIK, conversions for USDC are already configured in the converters. So, we shouldn't need it here again

Reply:
We have set a 0 conversion incentive on Sepolia for other tokens. Should we follow the same approach here as well? Currently, I am setting 1e14 as the conversion incentive.

Reply:
Ok, let's keep this command, setting 1e14